packages/react/src/generators/library/library.ts

```diff
@@ -61,6 +61,9 @@ export async function libraryGenerator(host: Tree, schema: Schema) {
       `For publishable libs you have to provide a proper "--importPath" which needs to be a valid npm package name (e.g. my-awesome-lib or @myorg/my-lib)`
     );
   }
+
```

Comment:
I added this comment as it seems like setting `none` would break some of the conventions below in this file. I know I have generated a library without an initial component as I wanted to use a custom generator not the default component generator.